apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -76,7 +76,7 @@ const ConfigScreen = () => {
 
     async function checkToken() {
       if (vercelClient) {
-        validateToken(updateTokenValidityState);
+        await validateToken(updateTokenValidityState);
```

Comment:
s/o to George for pointing out an obvious improvement here!!! 🙌  I knew something was not working quite right 😅